arbeitszeit_flask/templates/macros/draft_form.html

```diff
@@ -39,18 +36,21 @@
 </div>
 <div class="block py-2"></div>
 <div class="box has-background-info-light">
-  <div class="has-text-right has-text-info-dark js-modal-trigger" data-target="id_unit_amount">
-    <span class="icon"><i class="fa-solid fa-circle-question"></i></span>
-  </div>
   <div class="field">
+    <div class="has-text-right has-text-info-dark">
+      {{ info_tooltip(id="id_unit", text=gettext("This value is used for calculation purposes only, i.e. you do not have to adhere to any rules here. For example, if you plan to produce beverages, you are free to choose if your smallest delivery unit is one can or one six-pack of cans."))}}
```

Comment:
It is actually more of an "informational purpose", I would say (you inform the potential consumers). It is, by the way, also relevant for cooperations, where the delivery units should match (we do not enforce this, though). 

What about this?

"This value is used for informational purposes only, i.e. you do not have to adhere to any rules here. For example, if you plan to produce beverages, your smallest delivery unit might be one can or one six-pack of cans."

Reply:
done. The info about cooperations is maybe sth for the "help" docu?

Reply:
Yes, the whole cooperation thing is still missing there.